keras_cv/models/backbones/efficientnet_v2/efficientnet_v2_backbone_presets.py

```diff
@@ -0,0 +1,778 @@
+            "activation": "swish",
+            "include_rescaling": True,
+            "input_shape": (None, None, 3),
+            "input_tensor": None,
+            "block_args": B1_BLOCK_ARGS,
+        },
+    },
+    "efficientnetv2-b2": {
+        "metadata": {
+            "description": "The EfficientNet B2 architecture.  In this "
+            "variant of the EfficientNet architecture, there are "
+            f"{len(B2_BLOCK_ARGS)} convolutional blocks. As with all of the B "
+            "style EfficientNet variants, the number of filters in each "
+            "convolutional block is scaled by a `width_coefficient` and a "
+            "`depth_coefficient`.  Please see the GitHub source to find the "
+            "specific values for both the `width_coefficient` and "
+            "`depth_coefficient`"
+        },
+        "class_name": "keras_cv.models>EfficientNetV2Backbone",
+        "config": {
+            "width_coefficient": 1.1,
+            "depth_coefficient": 1.2,
+            "default_size": 260,
+            "dropout_rate": 0.2,
+            "drop_connect_rate": 0.2,
+            "depth_divisor": 8,
+            "min_depth": 8,
+            "activation": "swish",
+            "include_rescaling": True,
+            "input_shape": (None, None, 3),
+            "input_tensor": None,
+            "block_args": B2_BLOCK_ARGS,
+        },
+    },
+    "efficientnetv2-b3": {
+        "metadata": {
+            "description": "The EfficientNet B3 architecture.  In this "
+            "variant of the EfficientNet architecture, there are "
+            f"{len(B3_BLOCK_ARGS)} convolutional blocks. As with all of the B "
+            "style EfficientNet variants, the number of filters in each "
+            "convolutional block is scaled by a `width_coefficient` and a "
+            "`depth_coefficient`.  Please see the GitHub source to find the "
+            "specific values for both the `width_coefficient` and "
+            "`depth_coefficient`"
+        },
+        "class_name": "keras_cv.models>EfficientNetV2Backbone",
+        "config": {
+            "width_coefficient": 1.2,
+            "depth_coefficient": 1.4,
+            "default_size": 300,
+            "dropout_rate": 0.2,
+            "drop_connect_rate": 0.2,
+            "depth_divisor": 8,
+            "min_depth": 8,
+            "activation": "swish",
+            "include_rescaling": True,
+            "input_shape": (None, None, 3),
+            "input_tensor": None,
+            "block_args": B3_BLOCK_ARGS,
+        },
+    },
+}
+
+backbone_presets_with_weights = {
+    "efficientnetv2-s_imagenet": {
+        "metadata": {
+            "description": {
+                "description": "The EfficientNet small architecture.  In this "
+                "variant of the EfficientNet architecture, there are "
+                f"{len(SMALL_BLOCK_ARGS)} convolutional blocks. Weights are "
+                "initialized to pretrained imagenet classification weights."
```

Comment:
Do we want to state the scores of these pre-trained weights?

Reply:
(We do for OD models, probably should here too I think)

Reply:
Done!  Thanks